doc/nrf/drivers/sensor_stub.rst

```diff
@@ -27,12 +27,13 @@ You can define instances of the sensor stub on Devicethree like this:
         };
 
 The most important setting here is the ``generator`` property.
-The driver would require the user to implement following functions:
-- ``int <generator>_init(const struct device *dev)`` - initialize the driver
-- ``int <generator>_get(const struct device *dev, enum sensor_channel ch, struct sensor_value *val)`` - get the measure value
-- ``int <generator>_fetch(const struct device *dev, enum sensor_channel ch)`` - fetch (prepare) the measured value
+The driver would require the user to implement the following functions:
 
-The functions has to have global linkage - they would be connected directly in the sensor stub driver.
+- ``int <generator>_init(const struct device *dev)`` - The function initializes the driver.
+- ``int <generator>_get(const struct device *dev, enum sensor_channel ch, struct sensor_value *val)`` - To get the measured value.
+- ``int <generator>_fetch(const struct device *dev, enum sensor_channel ch)`` - To fetch (prepare) the measured value.
```

Comment:
```suggestion
* ``int <generator>_init(const struct device *dev)`` - This function initializes the driver.
* ``int <generator>_get(const struct device *dev, enum sensor_channel ch, struct sensor_value *val)`` - This function gets the measured value.
* ``int <generator>_fetch(const struct device *dev, enum sensor_channel ch)`` - This function fetches (prepares) the measured value.
```
Why is "prepares" mentioned in the last sentence?

Reply:
This module is used mainly to simulate sensor value. Then the value is rather prepared (calculated) than fetched. But it can be used for simplify some application sensor implementation as well.